__tests__/Unit/Components/Tasks/SuggestionBox.test.tsx

```diff
@@ -0,0 +1,55 @@
+import SuggestionBox from '@/components/tasks/SuggestionBox/SuggestionBox';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+
+const gitInfo: GithubInfo[] = [
+    {
+        github_id: 'Jan',
+        profileImageUrl: placeholderImageURL,
+    },
+    {
+        github_id: 'John',
+        profileImageUrl: placeholderImageURL,
+    },
+];
```

Comment:
can we take this into fixtures folder and from there import it,
remember we should try to put all the constants into one file so that our main file looks clean.

Reply:
@Pratiyushkumar Sorry, we cannot. as we have no fixture folder.

Reply:
__mocks__/db is your fixture folder, 
https://stackoverflow.com/questions/12071344/what-are-fixtures-in-programming

Reply:
> **mocks**/db is your fixture folder, https://stackoverflow.com/questions/12071344/what-are-fixtures-in-programming

Thank you, I know what fixtures are tho.